src/linker/pipeline_schema.py

```diff
@@ -11,6 +11,7 @@ class PipelineSchema:
     def __init__(self, name) -> None:
         self.name = name
         self.steps = []
+        self.validate_input: Callable = validate_dummy_input
```

Comment:
Part of why I did this was because I didn't want to have to figure out the right validation method for `pvs_like_case_study`. Are you OK with me just patching around it for now with a TODO?

Reply:
done